systolic/CgraSystolicArrayRTL.py

```diff
@@ -95,11 +93,10 @@ def construct(s, DataType, PredicateType, CtrlPktType, CtrlSignalType,
     s.data_mem.send_to_noc_load_response_pkt //= s.controller.recv_from_tile_load_response_pkt
     s.data_mem.send_to_noc_store_pkt //= s.controller.recv_from_tile_store_request_pkt
 
-    s.recv_from_noc //= s.controller.recv_from_noc
-    s.send_to_noc //= s.controller.send_to_noc
+    s.controller.send_to_noc //= s.controller.recv_from_noc

```

Comment:
Add comment above mentioning `# As we always first issue request pkt from CPU to NoC, when there is no NoC for single CGRA test, we have to connect from_noc and to_noc in testbench.`